### PR TITLE
`utilities` package

### PR DESCRIPTION
**Pull request by missingfaktor**
This is more in line with Scala conventions.

### CODE REVIEW

**Review by amarpotghan — APPROVED** [review]
better indeed.

File: src/main/scala/dhall/utilities/TypeLevelFunctions.scala [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package dhall.utilities
+
+import language.higherKinds
```
Comment (amarpotghan) [inline src/main/scala/dhall/utilities/TypeLevelFunctions.scala:3]: If we use higherKinds elsewhere as well then we should enable it from sbt I think.

Comment (missingfaktor) [reply src/main/scala/dhall/utilities/TypeLevelFunctions.scala:3]: Good thought. I will make that change.

Comment (missingfaktor) [reply src/main/scala/dhall/utilities/TypeLevelFunctions.scala:3]: I will just push that to master, since the two of us agree to it.

**Pull request merged**